spec/controllers/solidus_bolt/webhooks_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'SolidusBolt::WebhooksController', type: :request do
+  describe '#update' do
+    subject(:endpoint_call) { post "/webhooks/bolt", params: {} }
+
+    context "when valid" do
+      let(:expected_params) { ActionController::Parameters.new({}) }
+      let(:hash_header)     { nil }
+
+      before do
+        allow(::SolidusBolt::Webhooks::Sorter).to receive(:call)
+        endpoint_call
+        expected_params[:controller] = "solidus_bolt/webhooks"
+        expected_params[:action] = "update"
+      end
+
+      it "have http status success" do
+        expect(response).to have_http_status(:success)
+      end
+
+      it "calls the webhook sorter with the correct params" do
+        expect(SolidusBolt::Webhooks::Sorter).to have_received(:call).with(expected_params, hash_header)
```

Comment:
You could make it more clear(avoiding to the reader to scroll up and down) with `.with(hash_including(controller: 'solidus_bolt/webhooks', action: 'update'))`

Reply:
this however generates an error
```
         expected: (hash_including(:controller=>"solidus_bolt/webhooks", :action=>"update"))
              got: (#<ActionController::Parameters {"controller"=>"solidus_bolt/webhooks", "action"=>"update"} permitted: false>, nil)
```

Reply:
That makes sense 👍